<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="contaLbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Conta</string>
    </property>
   </widget>
   <widget class="QLabel" name="numeroLbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Número:</string>
    </property>
   </widget>
   <widget class="QLabel" name="clientLbl">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cliente:</string>
    </property>
   </widget>
   <widget class="QLabel" name="saldoInicialLbl">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Saldo Inicial:</string>
    </property>
   </widget>
   <widget class="QLabel" name="arquivoContaLbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Arquivo de Contas:</string>
    </property>
   </widget>
   <widget class="QLabel" name="arquivoTransacoesLbl">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Arquivo de Transações:</string>
    </property>
   </widget>
   <widget class="QLabel" name="contaPesquisadaLbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Número da Conta Pesquisada:</string>
    </property>
   </widget>
   <widget class="QLabel" name="transacoesPesquisadaLbl_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Transações da Conta Pesquisada:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="contasArquivoBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Abrir Arquivo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="transacoesArquivoBtn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Abrir Arquivo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="atualizarSaldoBtn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Atualizar Saldo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pesquisarTransacoesBtn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Pesquisar Transações</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cadastrarBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Cadastrar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numeroTxt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="clienteTxt">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>50</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="saldoInicialTxt">
    <property name="geometry">
     <rect>
      <x>332</x>
      <y>50</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="arquivoContaTxt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="arquivoTransacoesTxt">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="contaPesquisadaTxt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="transacoesPesquisadaTxt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>411</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cadastrarBtn_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Ler</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
